cedar-lean/Cedar/Thm/Data/List/Lemmas.lean

```diff
@@ -1202,3 +1202,16 @@ theorem length_removeAll_le {α : Type u_1} [BEq α] (xs ys : List α) :
   omega
 
 end List
+
+-- forM and mapM
+
+theorem forM_mapM {α β : Type} (f : α → Except β PUnit) (xs : List α) (ys : List PUnit) :
```

Comment:
Formatting nit: see [here](https://github.com/cedar-policy/cedar-spec/blob/main/cedar-lean/GUIDE.md) for a guide on indentation and formatting.

We would usually format this theorem statement so that the name of the theorem and arguments are on one line, the hypotheses and conclusion are indented and on separate lines, `:= by` is on its own line and not indented, and the proof body is indented.  For example:

```
theorem forM_mapM {α β : Type} (f : α → Except β PUnit) (xs : List α) (ys : List PUnit) :
  xs.mapM f = Except.ok ys → xs.forM f = Except.ok () 
:= by
  intro h₀
  rw [List.mapM_ok_iff_forall₂] at h₀
  sorry
```